openedx/core/djangoapps/user_api/urls.py

```diff
@@ -12,17 +12,22 @@
 urlpatterns = patterns(
     '',
     url(
-        r'^v1/accounts/' + USERNAME_PATTERN + '$',
+        r'^v1/accounts/%s$' % (USERNAME_PATTERN,),
```

Comment:
i believe our convention these days is to use [`str.format`](https://docs.python.org/2/library/stdtypes.html#str.format) rather than interpolation.  (applies to the other entries below as well)